cmd/root.go

```diff
@@ -40,6 +40,17 @@ func init() {
 
 	rootCmd.PersistentFlags().CountVarP(&persistentOpts.Verbosity, "verbose", "v", "increase verbosity (-v = info, -vv = debug)")
 
+	rootCmd.PersistentFlags().StringP(
+		"file", "f", "",
```

Comment:
Should we remove the `"f"` for now?

Reply:
this was my bad --I removed it but was restoring work from the reflog and accidentally brought it back. Removing again...